engine/main.py

```diff
@@ -640,6 +645,28 @@ def unsubscribe_group(group_name, user):
     logging.debug(res)
     return res
 
+def check_unsubscribe(membergroup, user):
+    """Checks whether or not a user can unsubscribe from a mailing list
+
+    Args:
+        membergroup(MemberGroup): a MemberGroup instance of a user who tries to unsubscribe
```

Comment:
it is a Queryset which contains lots of instances. So should be membergroup**s** (plural).